pdns/dnsdistdist/docs/advanced/tuning.rst

```diff
@@ -58,3 +63,18 @@ The UDP threads handling the responses from the backends do not use a lot of CPU
   newServer({address="192.0.2.127:53", name="Backend2"})
   newServer({address="192.0.2.127:53", name="Backend3"})
   newServer({address="192.0.2.127:53", name="Backend4"})
+
+For DNS over HTTPS, every :func:`addDOHLocal` directive adds a new thread dealing with incoming connections, so it might be useful to add more than one directive, as indicated above.
+
+When dealing with a large traffic load, it might happen that the internal pipe used to pass queries between the threads handling the incoming connections and the one getting a response from the backend become full too quickly, degrading performance and causing timeouts. This can be prevented by increasing the size of the internal pipe buffer, via the `internalPipeBufferSize` option of :func:`addDOHLocal`. Setting a value of `1048576` is known to yield good results on Linux.
+
+When dispatching UDP queries to backend servers, dnsdist keeps track of at most **n** outstanding queries for each backend.
+This number **n** can be tuned by the :func:`setMaxUDPOutstanding` directive, defaulting to 10240 (65535 since 1.4.0), with a maximum value of 65535.
```

Comment:
I would say "defaulting to 65535 (10240 before 1.4.0)" or even

```rst
defaulting to 65535, which is the maximum value.

.. versionchanged:: 1.4.0
  The default was 10240 before 1.4.0
```

Reply:
Applied, thanks!